src/GitHub.App/Models/PullRequestModel.cs

```diff
@@ -0,0 +1,111 @@
+﻿using System;
+using System.Globalization;
+using GitHub.Primitives;
+using GitHub.VisualStudio.Helpers;
+using NullGuard;
+
+namespace GitHub.Models
+{
+    public sealed class PullRequestModel : NotificationAwareObject, IPullRequestModel
+    {
+        public PullRequestModel(int number, string title, IAccount author, DateTimeOffset createdAt, DateTimeOffset? updatedAt = null)
+        {
+            Number = number;
+            Title = title;
+            Author = author;
+            CreatedAt = createdAt;
+            UpdatedAt = updatedAt ?? CreatedAt;
+        }
+
+        public void CopyFrom(IPullRequestModel other)
```

Comment:
This method is confusing. I was wondering why it doesn't update `Number` and `Author` and it occurred to me, this isn't a "copy" method, it's an "update" method, right? That's why it only updates fields that could possibly change for a given pull request.

Should we perhaps ensure that the `other.Id` matches the current `Id`?